app/javascript/src/components/editor/FindReplaceAll.svelte

```diff
@@ -115,7 +115,11 @@
     if (event.ctrlKey && event.shiftKey && event.keyCode == 70) { // F key
       event.preventDefault()
       active = !active
-      if (active) focusInput()
+      if (active) {
+        focusInput()
+        const codeMirrorSelectedText = $editorStates[$currentItem.id].sliceDoc($editorStates[$currentItem.id].selection.main.from, $editorStates[$currentItem.id].selection.main.to)
```

Comment:
Perhaps `$editorStates[$currentItem.id].selection.main.from, $editorStates[$currentItem.id].selection.main.to` could be more readable like
```suggestion
const [from, to] = $editorStates[$currentItem.id].selection.main
const codeMirrorSelectedText = $editorStates[$currentItem.id].sliceDoc(from, to)
```

Reply:
Oooh, I like it!